Directory.Build.targets

```diff
@@ -156,25 +154,27 @@ Expected to exist:
   <!-- Build the iOS SDK: dotnet msbuild /t:BuildiOSSDK src/Sentry.Unity -->
   <Target Name="BuildiOSSDK" Condition="$([MSBuild]::IsOSPlatform('OSX'))
           And '$(MSBuildProjectName)' == 'Sentry.Unity'
-          And (!Exists('$(SentryiOSDeviceArtifactsDestination)') OR !Exists('$(SentryiOSSimulatorArtifactsDestination)'))" BeforeTargets="BeforeBuild">
+          And (!Exists('$(SentryiOSArtifactsDestination)'))" BeforeTargets="BeforeBuild">
     <Error Condition="!Exists('$(SentryCocoaRoot)')" Text="Couldn't find the Cocoa root at $(SentryCocoaRoot)."></Error>
     <Message Importance="High" Text="Building Sentry iOS SDK."></Message>
 
-    <Exec WorkingDirectory="$(SentryCocoaRoot)" Command="carthage build --use-xcframeworks --no-skip-current --platform iOS"></Exec>
+    <!--    Workaround to enable building the Cocoa SDK with Bitcode enabled -->
+    <Message Importance="High" Text="Creating xcconfig file to enable Bitcode." />
+    <WriteLinesToFile File="$(SentryCocoaRoot)unity.xcconfig" Lines="ENABLE_BITCODE = YES" Overwrite="true" />
```

Comment:
The Cocoa builds with Bitcode `disabled` by default. Unity Xcode projects have Bitcode `enabled` by default. Bitcode has been deprecated but while we wait for Unity to update the generation we build the Cocoa SDK with a temporary config file to enable it.